vm/devices/vmbus/vmbus_server/src/lib.rs

```diff
@@ -958,6 +949,7 @@ impl ServerTask {
             VmbusRequest::Start => {
                 if !self.inner.running {
                     self.inner.running = true;
+                    self.server.with_notifier(&mut self.inner).post_restore();
```

Comment:
Rename `post_restore` to something more specific about what it does (e.g. `revoke_unclaimed_channels` or whatever).